pkg/cmd/issue/create/create.go

```diff
@@ -62,13 +64,22 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 			titleProvided := cmd.Flags().Changed("title")
 			bodyProvided := cmd.Flags().Changed("body")
 			opts.RepoOverride, _ = cmd.Flags().GetString("repo")
+			opts.JSONFill = cmd.Flags().Changed("json")
 
 			opts.Interactive = !(titleProvided && bodyProvided)
 
 			if opts.Interactive && !opts.IO.CanPrompt() {
 				return &cmdutil.FlagError{Err: errors.New("must provide --title and --body when not running interactively")}
 			}
 
+			if opts.JSONFill {
+				opts.Interactive = false
```

Comment:
this occurred to me, but I couldn't think of any examples of failure that wouldn't work with automatic resubmit. If a user manually bails out after filling in a body (because perhaps they wanted some more commits in), my thinking was that they could manually get data back out of the dump file in that case.

I'm open to retooling for "pick up where I left off" but I'm curious what @ampinsk thinks about it

Reply:
@vilmibm would it be a lot of work to retool for interactive resubmission?

Reply:
not a ton, nope

Reply:
if it's not then I think that would be a nice touch!

Reply:
I'll work on that today.

Reply:
This is in, now; I definitely think it's a better experience.